lib/mindwendel/brainstormings/brainstorming.ex

```diff
@@ -43,4 +55,14 @@ defmodule Mindwendel.Brainstormings.Brainstorming do
     |> change(%{})
     |> put_assoc(:users, users)
   end
+
+  def idea_label_factory do
```

Comment:
i would move this into the label module - or why is it placed here?

Reply:
It is defined in the brainstorming module because the Brainstorming defines its default set of idea labels and not the other way around.